com.unity.build-report-inspector/Editor/BuildReportInspector/Mobile/MobileHelper.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using UnityEngine;
+
+namespace Unity.BuildReportInspector.Mobile
+{
+    internal static class MobileHelper
+    {
+        internal static IPlatformUtilities s_PlatformUtilities;
+        internal static string AppendixSavePath => Path.Combine(Directory.GetParent(Application.dataPath).FullName, "Library/MobileReports");
+
+        internal static void RegisterPlatformUtilities(IPlatformUtilities utilities)
+        {
+            if (!Directory.Exists(AppendixSavePath))
+                Directory.CreateDirectory(AppendixSavePath);
+
+            if (s_PlatformUtilities != null)
```

Comment:
Curious about what will happen if you switch between iOS and Android platforms.

Reply:
Nothing, the s_PlatformUtilities is lost upon domain reload and re-initialized.

Reply:
Ah, yes, forgot this..